README.md

```diff
@@ -60,7 +60,7 @@ For prerequisites, installation and setup, see the [Installation and setup](docs
 Currently the Unity GDK is in its early development, so there is no migration path from the [Unity SDK](https://github.com/spatialos/UnitySDK) to the Unity GDK. There will be a migration path in later releases.
 
 ## Roadmap
-See the latest Roadmap [here](docs/Roadmap.md) and don't hesitate to feedback!
+See the [latest roadmap](docs/roadmap.md) and let us know what you'd like to see in there.
```

Comment:
The link doesn't work for me unless I change `roadmap` to `Roadmap` - do you need to rename `docs/Roadmap.md`?

Reply:
yeah, you need to keep the exact same names for the link

Reply:
I am updating the 'R' to 'r' in this PR which is why it won't work if you view the page from this PR - it points to the existing one.

Reply:
I think you either need to change the file name (Roadmap.md) to lower case in this PR as well, or you need to keep the link as "Roadmap.md" (but obviously the link text can have lower case). Does that make sense?